docs/01-Introduction.md

```diff
@@ -1,6 +1,82 @@
 # 1.0 - Introduction
 
-## 1.1 What is LORIS?
+### 1.1 What is LORIS-MRI?  LORIS-MRI is the backbone of the imaging
+component that makes up LORIS.
+ It is maintained in a separate repository so that it can be installed
+ on the file server and separated from the web server.
+These documents assume you have some knowledge regarding LORIS and
+a functioning installation. For information regarding LORIS itself,
+please consult the [LORIS wiki][1].
 
-## 1.2 How does LORIS work?
+LORIS-MRI is a set of libraries responsible for the insertion,
+visualization and archival of uploaded imaging datasets.
+ It expects an uploaded, compressed file containing
+several [DICOM][2] files, converts this into [MINC][3] and [NII][4]
+files, and then archives these files onto the server. Knowledge of
+these file formats can be helpful, but are not necessary for using or
+installing LORIS-MRI.
 
+LORIS-MRI allows you to easily organize and archive your imaging datasets
+and links them with
+ corresponding behavioral data in LORIS. Scans can be viewed and quality
+ controlled through the user's web browser, facilitating collaboration
+ between radiologists, clinicians and researchers.
+
+### 1.2 How does LORIS-MRI work?  ![user_story](images/user_story.png)
+LORIS-MRI allows multiple ways to upload scans, more details about these
+options can be found [here.](05-PipelineLaunchOptions.md) First, users
+upload a compressed (.tgz, .tar.gz, or .zip) DICOM folder to Imaging
+Uploader.  LORIS requires that the uploaded file name follow the naming
+convention PSCID_CANDID_VISIT-LABEL.  Assuming the upload went through
+successfully, an entry is added to the menu of Imaging Uploader and the
+upload's status throughout the pipeline can then be tracked by the user
+through the Log Viewer.
+
+All DICOM datasets transferred to the DCC must be free of any identifying
+information (e.g. patient name). A tool can be provided to the sites to
+facilitate deidentification. Please contact the LORIS team for details.
+
+The LORIS-MRI pipeline starts once the scans are uploaded to the server.
+The pipeline can start automatically if the autolaunch configuration is
+set, otherwise
+ manual intervention is required by a developer with backend access to
+ the server (information on the scripts needed to run the pipeline can
+ be found in the [Scripts section](04-Scripts.md) of the documentation).
+
+ First, the upload is prepped to be archived on the server. The
+ PatientName in the
+  DICOM headers is validated by the LORIS database and the upload is
+  checked to ensure that only DICOM files were uploaded.  Given that
+ all uploaded metadata is valid, the DICOM upload is archived on the
+ server.
+
+ Following archival, the scans are converted into MINC and (optionally)
+ NII file formats. At this stage, scan parameters such as echo time,
+ repetition time, etc., are checked with the MRI protocol table in the
+ LORIS database to determine the scan types.
```

Comment:
Add the following at the end of this paragraph:
"Once the scan type has been determined, the scan are inserted into the database and viewable via the Imaging Browser module. If the scan type could not be determined, then the scan will be accessible via the MRI Violated Scans module."